src/Microsoft.ML.Transforms/MutualInformationFeatureSelection.cs

```diff
@@ -41,19 +48,45 @@ public sealed class Arguments : TransformInputBase
 
             [Argument(ArgumentType.LastOccurenceWins, HelpText = "Column to use for labels", ShortName = "lab",
                 SortOrder = 4, Purpose = SpecialPurpose.ColumnName)]
-            public string LabelColumn = DefaultColumnNames.Label;
+            public string LabelColumn = Defaults.LabelColumn;
 
             [Argument(ArgumentType.AtMostOnce, HelpText = "The maximum number of slots to preserve in output", ShortName = "topk,numSlotsToKeep",
                 SortOrder = 1)]
-            public int SlotsInOutput = 1000;
+            public int SlotsInOutput = Defaults.SlotsInOutput;
 
             [Argument(ArgumentType.AtMostOnce, HelpText = "Max number of bins for R4/R8 columns, power of 2 recommended",
                 ShortName = "bins")]
-            public int NumBins = 256;
+            public int NumBins = Defaults.NumBins;
         }
 
         internal static string RegistrationName = "MutualInformationFeatureSelectionTransform";
 
+        /// <summary>
+        /// A helper method to create <see cref="MutualInformationFeatureSelectionTransform"/> for public facing API.
```

Comment:
>MutualInformationFeatureSelectionTransform [](start = 49, length = 42)

You can't be creating it, it's a static class.

Reply:
Similar comments as before, when documenting we ought to describe what it is actually useful for, not just describing its return values (which are part of the method signature, and we aren't even doing that quite right).

---
In reply to: [201912038](https://github.com/dotnet/machinelearning/pull/520#discussion_r201912038) [](ancestors = 201912038)

Reply:
I will add detailed comments as part of  #524.

---
In reply to: [201912123](https://github.com/dotnet/machinelearning/pull/520#discussion_r201912123) [](ancestors = 201912123,201912038)